examples/resources/vmp_custom_rule_set/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "waf_custom_rule_id" {
+  description = "custom_rule_id"
+  value       = vmp_waf_custom_rule.custom_rule_1.*.id
```

Comment:
Please replace vmp references in the folder structure and output to ec

Reply:
Updated. Thanks.